qaframework-bdd-tests/src/test/resources/features/refapp-2.x/07-visit/visitTypes.feature

```diff
@@ -1 +1,33 @@
-Feature: Visit types Management
\ No newline at end of file
+Feature: Visit types Management
+
+  Background:
+    Given a user clicks on the system administration link from the home page
+    And the system loads the administration page and User clicks on the manage Visit Types link
```

Comment:
Its a good idea to have each step captured separately than combining two steps in one.